packages/commonwealth/server/models/profile.ts

```diff
@@ -52,6 +53,7 @@ export default (
         primaryKey: true,
       },
       user_id: { type: dataTypes.INTEGER, allowNull: false },
+      username: { type: dataTypes.STRING, allowNull: false, unique: true },
```

Comment:
To enforce this, we will need to add generated profile names to each currently null "username" field on existing profiles. We will then need to define the "unique" constraint in the migration itself.

Reply:
Zak and I had discussed that adding generated usernames would be done in a later PR: https://www.notion.so/hicommonwealth/PR-4-Profiles-Conflict-Resolution-Jesse-to-Get-Numbers-on-Jake-Zak-b3b92e05b092470781569ce525dfffa0

@zakhap is this still the case?